src/System.IO.Pipes/tests/PipeTest.Write.cs

```diff
@@ -14,214 +14,246 @@ namespace System.IO.Pipes.Tests
     /// </summary>
     public abstract class PipeTest_Write : PipeTestBase
     {
+        protected virtual bool ShouldSkipTest() => false;
+
         [Fact]
         public void WriteWithNullBuffer_Throws_ArgumentNullException()
         {
-            using (ServerClientPair pair = CreateServerClientPair())
+            if (!ShouldSkipTest())
```

Comment:
No need to change now, but in general, I prefer the style of 
```c#
public void WriteWithNullBuffer_Throws_ArgumentNullException()
{
        if (ShouldSkipTest())
               return;
   ...
```

ie., where an if wraps the whole body of the method, return early instead. This avoids "triangle code" and in this case also avoids a double negative which are often a source of bugs.

Reply:
It also would greatly reduce the diff :)